pxr/usd/sdf/textParserContext.h

```diff
@@ -47,8 +47,6 @@
 // Lexical scanner type.
 typedef void *yyscan_t;
 
-PXR_NAMESPACE_OPEN_SCOPE
```

Comment:
There was some refactoring done to make this object useable from both lexers. If I remember correctly, removing the namespace here made the resulting .lex files much simpler (because they don't live in the namespace) but I'd have to back and verify.

Reply:
Placed them back under the namespace and verified behavior.